src/art/skypilot/api.py

```diff
@@ -34,7 +36,7 @@ async def initialize_cluster(
     ) -> None:
         self = cls.__new__(cls)
         self._cluster_name = cluster_name
-
+        self._job_id = None
```

Comment:
Could we rename this to `_art_server_job_id`? We may have more tasks in the future, and I can imagine myself forgetting whether this id refers to the task of launching the cluster as a whole or the art server in particular.